changebot/webapp.py

```diff
@@ -85,7 +85,11 @@ def hook():
                'a maintainer you can ignore this, and a maintainer will let '
                'you know if any action is required on your part :smiley:.\n\n')
 
-    if len(issues) > 0:
+    if 'Work in progress' in pr_handler.labels:
+        message += ("I see this is a work in progress pull request. I'll "
+                    "carry out the checks once the PR is ready for review.")
```

Comment:
maybe say "report back" rather than "carry out"? As technically the checks are done just listed in the comment